sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -382,10 +382,10 @@ export class BlobClient extends StorageClient {
     createSnapshot(options?: BlobCreateSnapshotOptions): Promise<BlobCreateSnapshotResponse>;
     delete(options?: BlobDeleteOptions): Promise<BlobDeleteResponse>;
     deleteIfExists(options?: BlobDeleteOptions): Promise<BlobDeleteIfExistsResponse>;
-    download(offset?: number, count?: number, options?: BlobDownloadOptions): Promise<BlobDownloadResponseModel>;
+    download(offset?: number, count?: number, options?: BlobDownloadOptions): Promise<BlobDownloadResponseParsed>;
```

Comment:
Is this tageted to align with other langs? What about BlobDownloadResponse?

Reply:
`BlobDownloadResponse` is already defined in src\BlobDownloadResponse.ts.
.NET has different wrapping, returning `Response<BlobDownloadInfo>`.